dev/NavigationView/NavigationView.xaml

```diff
@@ -452,12 +443,15 @@
                                             VerticalContentAlignment="Stretch"
                                             HorizontalContentAlignment="Stretch"
                                             Grid.Row="7" />
-                                        <Grid
+                                        <!-- Left footer menu ItemsRepeater -->
+                                        <local:ItemsRepeater
+                                            x:Name="FooterMenuItemsHost"
+                                            AutomationProperties.AccessibilityView = "Content"
                                             Grid.Row="8">
-                                            <local:NavigationViewItem
-                                            x:Name="SettingsNavPaneItem"
-                                            Icon="Setting"/>
-                                        </Grid>
+                                            <local:ItemsRepeater.Layout>
+                                                <local:StackLayout Orientation="Vertical"/>
+                                            </local:ItemsRepeater.Layout>
```

Comment:
This is not needed. The default layout is vertical stack.

Reply:
Should i also remove it from  ```MenuItemsHost```?

Reply:
Yes. we can. 👍